stravalib/model.py

```diff
@@ -164,43 +219,89 @@ def deserialize(cls, attribute_value_mapping: Dict):
         )
         return cls.parse_obj(attribute_value_mapping)
 
-    def from_dict(self, attribute_value_mapping: Dict):
+    def from_dict(self, attribute_value_mapping: Dict) -> None:
         """
-        Deserializes v into self, resetting and ond/or overwriting existing fields
+        Deserializes v into self, resetting and ond/or overwriting existing
+        fields
+
+        Parameters
+        ----------
+        attribute_value_mapping : Dict
+            A dictionary that will be deserialized into the parent object.
+
+        Deprecated
+        ----------
+        1.0.0
+            The `from_dict()` method is deprecated in favor of `parse_obj()` method.
+            For more details, refer to the Pydantic documentation:
+            https://docs.pydantic.dev/usage/models/#helper-functions
         """
         warn_method_deprecation(
             self.__class__.__name__,
             "from_dict()",
             "parse_obj()",
             "https://docs.pydantic.dev/usage/models/#helper-functions",
         )
-        # Ugly hack is necessary because parse_obj does not behave in-place but returns a new object
+        # Ugly hack is necessary because parse_obj does not behave in-place but
+        # returns a new object
         self.__init__(**self.parse_obj(attribute_value_mapping).dict())
 
-    def to_dict(self):
+    def to_dict(self) -> Dict:
         """
         Returns a dict representation of self
+
+        Returns
+        -------
+        Dict
+            A dictionary containing the data from the instance.
+
+        Deprecated
+        ----------
+            The `to_dict()` method is deprecated in favor of `dict()` method.
+            For more details, refer to the Pydantic documentation:
+            https://docs.pydantic.dev/1.10/usage/exporting_models/
         """
         warn_method_deprecation(
             self.__class__.__name__,
             "to_dict()",
             "dict()",
-            "https://docs.pydantic.dev/usage/exporting_models/",
+            "https://docs.pydantic.dev/1.10/usage/exporting_models/",
```

Comment:
this is just a page that no longer exists in the latest version of the docs so i pinned it to 1.10 where it does exist